library/src/main/java/com/novoda/downloadmanager/lib/BatchCompletionListener.java

```diff
@@ -0,0 +1,5 @@
+package com.novoda.downloadmanager.lib;
+
+public interface BatchCompletionListener {
```

Comment:
Not used any more https://github.com/novoda/download-manager/commit/8ae64aacecf611f3dc15c294fd5a58958e63157b